tests/baselines/reference/invalidTaggedTemplateEscapeSequences.errors.txt

```diff
@@ -0,0 +1,47 @@
+tests/cases/conformance/es2018/invalidTaggedTemplateEscapeSequences.ts(5,18): error TS1125: Hexadecimal digit expected.

```

Comment:
We still need to figure out exactly how to prevent the errors from the initial scan...